includes/Abilities/Image/Import.php

```diff
@@ -0,0 +1,273 @@
+<?php
+/**
+ * Image import WordPress Ability implementation.
+ *
+ * @package WordPress\AI
+ */
+
+declare( strict_types=1 );
+
+namespace WordPress\AI\Abilities\Image;
+
+use Throwable;
+use WP_Error;
+use WordPress\AI\Abstracts\Abstract_Ability;
+use WordPress\AiClient\Files\DTO\File;
+
+/**
+ * Image import WordPress Ability.
+ *
+ * @since 0.1.0
+ */
+class Import extends Abstract_Ability {
+
+	/**
+	 * {@inheritDoc}
+	 *
+	 * @since x.x.x
+	 */
+	protected function input_schema(): array {
+		return array(
+			'type'       => 'object',
+			'properties' => array(
+				'data'        => array(
+					'type'              => 'string',
+					'sanitize_callback' => 'sanitize_text_field',
+					'description'       => esc_html__( 'The base64 encoded image data to import into the media library.', 'ai' ),
+				),
+				'filename'    => array(
+					'type'              => 'string',
+					'sanitize_callback' => 'sanitize_text_field',
+					'description'       => esc_html__( 'The filename of the image.', 'ai' ),
+				),
+				'title'       => array(
+					'type'              => 'string',
+					'sanitize_callback' => 'sanitize_text_field',
+					'description'       => esc_html__( 'The title of the image.', 'ai' ),
+				),
+				'description' => array(
+					'type'              => 'string',
+					'sanitize_callback' => 'sanitize_text_field',
+					'description'       => esc_html__( 'The description of the image.', 'ai' ),
+				),
+				'mime_type'   => array(
+					'type'              => 'string',
+					'sanitize_callback' => 'sanitize_text_field',
+					'description'       => esc_html__( 'The MIME type of the image.', 'ai' ),
+				),
```

Comment:
Let's make this optional. If the `data` parameter is a Data URL, it will be this structure and contain the mime type:
```
data:[mime-type];base64,[base64-encoded-bytes]
```

If the mime type is not explicitly set and it's not a Data URL, then we can throw an error.

Reply:
I've set the default of that to `null` now so you can optionally pass this if you want: https://github.com/WordPress/ai/pull/134/changes/8b93cea71dea126e47d523e70172e797f1bbb7d9